docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/BootstrapSample.cs

```diff
@@ -12,7 +12,7 @@ public static void Example()
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
-            IEnumerable<SamplesUtils.DatasetUtils.BinaryLabelFloatFeatureVectorSample> enumerableOfData = SamplesUtils.DatasetUtils.GenerateBinaryLabelFloatFeatureVectorSamples(5);
+            IEnumerable<SamplesUtils.DatasetUtils.BinaryLabelFloatFeatureVectorFloatWeightSample> enumerableOfData = SamplesUtils.DatasetUtils.GenerateBinaryLabelFloatFeatureVectorFloatWeightSamples(5);
```

Comment:
How do you examine the effect of having weight? Should it be somehow checked in a test?

Reply:
Thats what i did in the test `SdcaLogisticRegressionWithWeight` .. Added two  trainers w/wo weights and verified it produced different metrics.  Is that sufficient ?

---
In reply to: [270204851](https://github.com/dotnet/machinelearning/pull/3104#discussion_r270204851) [](ancestors = 270204851)

Reply:
Their scores are similar. I'd like to have a more strict criterion. As you heard from Zeeshan S, tiny changes induced large SDCA regression this morning.

---
In reply to: [270205745](https://github.com/dotnet/machinelearning/pull/3104#discussion_r270205745) [](ancestors = 270205745,270204851)

Reply:
added checks in the other tests where we test thoroughly for this

---
In reply to: [270207991](https://github.com/dotnet/machinelearning/pull/3104#discussion_r270207991) [](ancestors = 270207991,270205745,270204851)